Lesson_8_Dictionary/Homework_8.py

```diff
@@ -0,0 +1,38 @@
+# Task_1 Hello Mr. key
+roles = { 'admin' : ['Evgeniy', 'Dmitry', 'asd'], 'maintainer' : ["Marina", "Sergey"],
+          'manager' : ["Anastasia", "Dmitry", 'asd'], 'developer': ["Evgeniy"]}
+name = input('What is yor name?')
+name_roles = []
+for key, value in roles.items():
+    if name in value:
+        name_roles.append(key)
+if name_roles:
+    print(f'Hello, {' & ' .join(name_roles)}!')
+else:
+    print('Hello, Guest!')
+
+# Task_2 add dictionary to string
+params = {'v': 'some_id', 'list': 'some_list', 'index': '6', 't': '0s'}
+initial_str = 'https://www.youtube.com/watch?'
+result_link = initial_str
+for key, values in params.items():
+    result_link += f'{key}={values}&'
+result_link = result_link[:-1:]
+print(result_link)
```

Comment:
You should use formatting string all time

Reply:
will correct